sql/dql/basic-selects/select-all.md

```diff
@@ -31,14 +31,30 @@ aspects:
 ---
 ## Content
 
-To get all of the data from a table, use a `SELECT *` statement.
+To retrieve data from a table we usually write queries that contain at least one `SELECT` statement. A query is just a way to declare the data that you want returned and where to find it in the database. On top of that, you can specify how you want your data displayed before it is returned but for this insight we will focus on the basics of queries.
```

Comment:
> queries that contain at least one `SELECT` statement

you can't write queries with more than one `SELECT`

"we write queries with the help of the `SELECT` statement"

Reply:
what about subqueries?